yaormfilter/boolfilter.go

```diff
@@ -0,0 +1,72 @@
+package yaormfilter
+
+import (
+	"github.com/geoffreybauduin/yaorm/tools"
```

Comment:
can you separate golang packages from community packages with a blank line?

Reply:
Doing now...